runtime/src/main/java/org/corfudb/runtime/view/StreamsView.java

```diff
@@ -97,6 +97,18 @@ public IStreamView copy(UUID source, UUID destination, long timestamp) {
         return get(destination);
     }
 
+    /** Append to multiple streams conditionally.
+     *
+     * @param conflictInfo                  Conflict information for conditional append.
+     * @param object                        The object to append.
+     * @return                              The address the object was appended at.
+     * @throws TransactionAbortedException  If the transaction was aborted by the sequencer.
+     */
+    public long conditionalAppend(@Nonnull TxResolutionInfo conflictInfo,
+                                  @Nonnull Object object) throws TransactionAbortedException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.corfudb.runtime.exceptions.TransactionAbortedException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)